apollo-router/src/services/supergraph_service.rs

```diff
@@ -408,23 +410,35 @@ impl RouterCreator {
         Error = BoxError,
         Future = BoxFuture<'static, Result<SupergraphResponse, BoxError>>,
     > + Send {
+        let supergraph_service = SupergraphService::builder()
+            .query_planner_service(self.query_planner_service.clone())
+            .execution_service_factory(ExecutionCreator {
+                schema: self.schema.clone(),
+                plugins: self.plugins.clone(),
+                subgraph_creator: self.subgraph_creator.clone(),
+            })
+            .schema(self.schema.clone())
+            .build();
+
+        let supergraph_service = match self
+            .plugins
+            .iter()
+            .find(|i| i.0.as_str() == "apollo.traffic_shaping")
```

Comment:
As you're using `apollo.traffic_shaping` at different places, could we put this in a constant ? To get rid of typo and so one

Reply:
fixed in https://github.com/apollographql/router/pull/2034/commits/037052927d4401471a1a66400e69f2795863114f